stablehlo/reference/Element.h

```diff
@@ -109,6 +109,9 @@ Element cosine(const Element &e);
 /// Returns element-wise floor of Element object.
 Element floor(const Element &e);
 
+/// Returns element-wise exponential of Element object.
+Element exponential(const Element &el);
+
```

Comment:
Let's order alphabetically - here and in the .cpp file.

Reply:
I'm mislead by the `cosine()` in Element.cpp which is out of order. 
Fixed `exponential` here. Not touch `cosine` in case conflict when merge.